cicd/gitops/using-argo-rollouts-for-progressive-deployment-delivery.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: ASSEMBLY
+[id="using-argo-rollouts-for-progressive-deployment-delivery"]
+= Using Argo Rollouts for progressive deployment delivery
+include::_attributes/common-attributes.adoc[]
+:context: using-argo-rollouts-for-progressive-deployment-delivery
+
+toc::[]
+
+:FeatureName: Argo Rollouts
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Progressive delivery is the process of releasing product updates in a controlled and gradual manner.
```

Comment:
(Applies to this entire introductory section) Should most of this text be moved to the "Benefits of Argo Rollouts" module?

Reply:
Nope. Not required.